pkg/crc/services/dns/dns_darwin.go

```diff
@@ -28,8 +28,15 @@ type resolverFileValues struct {
 }
 
 func runPostStartForOS(serviceConfig services.ServicePostStartConfig, result *services.ServicePostStartResult) (services.ServicePostStartResult, error) {
-	// Write resolver file
-	success, err := createResolverFile(serviceConfig.IP, filepath.Join("/", "etc", "resolver", serviceConfig.BundleMetadata.ClusterInfo.BaseDomain))
+	// Update resolv.conf file for host
+	success, err := updateResolvConfFile(serviceConfig.IP, filepath.Join("/", "etc", "resolv.conf"))
```

Comment:
Is there a bug filed against `oc` to ask them to use the native macos APIs? (though I suspect this comes from go stdlib).

Reply:
No, I didn't file it but as you said it is going to come from go stdlib, not sure if filling the issue there will make any sense.

Reply:
On the other hand, adding a workaround in our code without noticing other parties of the bug means this will never get fixed in the right place :(

Reply:
@cfergeau it is not only the `oc` even dig doesn't honour the resolver for mac, but I do get your point will try to create an issue on origin side.

Reply:
On my machine, it's only `oc` and `dig` which are using `resolv.conf`, `ping` for example does not seem to rely on it.